numpy/lib/_arraypad_impl.pyi

```diff
@@ -44,10 +44,16 @@ _ModeKind: TypeAlias = L[
 # specific modes. Consider adding more overloads to express this in the future.
 
 # Expand `**kwargs` into explicit keyword-only arguments
```

Comment:
this comment applies to the `pad` function, not the type alias, so the type alias should be above here